### PR TITLE
Library docstrings

### PR DESCRIPTION
**Pull request by JamesKunstle**
Adds docstrings for top-level job-queue and database manager objects.

### CODE REVIEW

**Comment by JamesKunstle** [comment]
Two separate commits because two separate changes.

File: job_manager/job_manager.py [hunk 1]
```diff
@@ -10,6 +10,44 @@
...
+            Manager for Redis in-memory cache that hosts queue for RQ workers
+            and RQ worker results.
+
+        _rq : (private) Queue object
+            Queue onto which jobs are pushed, FIFO, for RQ workers to pick up.
+
+        
+    Methods
+    -------
+        _get_job_hash(func, arglist) (private) :
```
Comment (cdolfi) [inline job_manager/job_manager.py:29]: wdym when you list things as private?

File: pages/utils/graph_utils.py [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
     period = "Y"
 
     # graph input values based on date interval selection
-    if interval == 86400000:  # if statement for days
-        x_r = [str(today - dt.timedelta(weeks=4)), str(today)]
-        x_name = "Day"
-        hover = "Day: %{x|%b %d, %Y}"
-        period = "D"
-    elif interval == "D":  # if statement for days
+    if interval == 86400000 or interval == "D":  # if statement for days
```
Comment (cdolfi) [inline pages/utils/graph_utils.py:29]: Did you test if this works? When I did it like this it was not producing the right results in all cases

File: pages/utils/graph_utils.py [hunk 3]
```diff
@@ -25,14 +36,10 @@ def get_graph_time_values(interval):
         x_name = "Week"
         hover = "Week: %{x|%b %d, %Y}"
         period = "W"
-    elif interval == "M":  # if statement for months
-        x_r = [str(today - dt.timedelta(weeks=104)), str(today)]
-        x_name = "Month"
-        hover = "Month: %{x|%b %Y}"
-        period = "M"
-    elif interval == "M1":  # if statement for months
+    elif interval == "M" or interval == "M1":  # if statement for months
```
Comment (cdolfi) [inline pages/utils/graph_utils.py:39]: same here

File: queries/commits_query.py [hunk 4]
```diff
@@ -3,14 +3,20 @@
 from db_manager.AugurInterface import AugurInterface
 
 
-def commits_query(dbmc, repo_ids):
+def commits_query(dbmc: AugurInterface, repo_ids: list(str)) -> dict:
```
Comment (cdolfi) [inline queries/commits_query.py:6]: Why is this one different than the other queries?

**Review by cdolfi — CHANGES_REQUESTED** [review]

File: job_manager/job_manager.py (hunk 1 above)
Comment (JamesKunstle) [reply job_manager/job_manager.py:29]: ah, good question. this means that it's not meant to be exposed to the user for use. python doesn't support real private/public interfaces like Java does so the underscore means that I won't necessarily want users touching that part of the method.

File: pages/utils/graph_utils.py (hunk 2 above)
Comment (JamesKunstle) [reply pages/utils/graph_utils.py:29]: haven't tested it, I'll check to see if it seems to comply on my end.

File: pages/utils/graph_utils.py (hunk 3 above)
Comment (JamesKunstle) [reply pages/utils/graph_utils.py:39]: (same as above)
haven't tested it, I'll check to see if it seems to comply on my end.

File: queries/commits_query.py (hunk 4 above)
Comment (JamesKunstle) [reply queries/commits_query.py:6]: ah good question, that was the first one I updated, forgot to do this to the other queries as well. these type annotations are "nice to have"s in that they're not enforced strictly like they are in other languages, but I need to also be consistent.

File: pages/utils/graph_utils.py (hunk 2 above)
Comment (cdolfi) [reply pages/utils/graph_utils.py:29]: @JamesKunstle will mark as resolved once its checked into

File: queries/commits_query.py (hunk 4 above)
Comment (cdolfi) [reply queries/commits_query.py:6]: @JamesKunstle will mark as resolved once the consistency is there

File: pages/utils/graph_utils.py (hunk 2 above)
Comment (JamesKunstle) [reply pages/utils/graph_utils.py:29]: I don't see any problems by doing this, the graphs look the same across the two implementations.

Comment (cdolfi) [reply pages/utils/graph_utils.py:29]: once the changes for the piplock is in ill test it out on my side too just to make sure. Just remember this being a niche issue and want to make 100% sure everything is good

File: queries/commits_query.py (hunk 4 above)
Comment (cdolfi) [reply queries/commits_query.py]: on my side I still see (dbmc: AugurInterface, repo_ids: list(str)) -> dict: for the commits query and (dbmc, repo_ids): for the rest

Comment (JamesKunstle) [reply queries/commits_query.py]: It was changed in the most recent commit to this branch, it's in files changed.

Comment (JamesKunstle) [reply queries/commits_query.py]: checked on it, up to date.

**Review by cdolfi — APPROVED** [review]

**Pull request merged**